spec/features/hub/show_client_spec.rb

```diff
@@ -6,7 +6,9 @@
     let(:client) { create :client, vita_partner: (create :vita_partner), intake: create(:intake, :with_contact_info), tax_returns: [create(:tax_return, certification_level: "advanced")] }
     let(:tax_return) { client.tax_returns.first }
     let!(:other_vita_partner) { create :vita_partner, name: "Tax Help Test" }
-    before { login_as user }
+    before do
+      login_as user
```

Comment:
This is weird; is this `login_as` unused? Best to move it into a `scenario` probably